cmd/neofs-cli/internal/commonflags/flags.go

```diff
@@ -49,6 +49,8 @@ const (
 // - Account
 // - RPC
 // - Timeout
+//
+//nolint:godot
```

Comment:
i think we can add `;` and `.` to that list. or the linter requires dots on every line? same for other cases

Reply:
No, it requires dots in the end only.
Fixed it here.